packages/block/src/header.ts

```diff
@@ -225,6 +227,17 @@ export class BlockHeader {
       }
     }
 
+    if (this._common.hardfork === Hardfork.Merge) {
```

Comment:
This works until we have a new hardfork after the Merge (i.e. Shanghai). It would be nice to have something on Common to figure out if the Merge is active or not.

Reply:
Good point. Will think of something better, thanks.

Reply:
We could use common.isSupportedEIP(3675)